src/services/calendar/index.js

```diff
@@ -216,12 +214,16 @@ class Service {
 		};
 
 		return request(options).then((events) => {
-			events = (params.query || {}).userId || (events.data || events || []).map((event) => Object.assign(event, {
-				title: event.summary,
-				allDay: false, // TODO: find correct value
-				start: Date.parse(event.dtstart),
-				end: Date.parse(event.dtend),
-			}));
+			events =
```

Comment:
Codacy found an issue: [Assignment to function parameter 'events'.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045788)